flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/DefaultOperatorCoordinatorHandlerTest.java

```diff
@@ -51,6 +55,12 @@ public class DefaultOperatorCoordinatorHandlerTest {
     public static final TestExecutorResource<ScheduledExecutorService> EXECUTOR_RESOURCE =
             TestingUtils.defaultExecutorResource();
 
+    private MetricRegistryImpl registry =
+            new MetricRegistryImpl(
+                    MetricRegistryTestUtils.defaultMetricRegistryConfiguration());
+    private final JobManagerMetricGroup jobManagerMetricGroup =
```

Comment:
You can simply return an `UnregisteredJobManagerJobMetricGroup` for testing.
`MetricRegistryImpl` is heavy and must be closed after the testing.